app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,22 @@
 class PostsController < ApplicationController
   def index
-    render 'index'
+    @posts = Post.new.print
+    @users = User.new.info(params[:user_id])
+    # @post_comments
   end
 
+  def comments(id)
+    Post.new.recent_comments(id)
+  end
+
+  def comments_all(id)
+    Post.new.recent_comments_all(id)
+  end
+
+  helper_method :comments, :comments_all
+
   def show
-    render 'show'
+    @post_show = Post.new.info(params[:id])
+    @user_show = User.new.info(params[:user_id])
   end
```

Comment:
- [ ] Kindly in Ruby on rails the syntax for getting info in a database table we use all methods on the modal. like `User.all`. and to find a specific user we use user.find(user_id). in your case you should use :point_down: 

```rb
 def index
    @ = Post.all
    @user = User.find(params[:user_id])
end
```
- [ ] After this action :point_up: ,make sure the correspondent view is working .

Reply:
This issue is well resolved! ✔️